README.md

```diff
@@ -31,13 +31,118 @@ To get started with development:
+
+- **Spacing, colors and font sizes**
+
+Using presets for spacing, font sizes, and colors in WordPress block patterns is preferred over hardcoded values for three key reasons:
+
+Consistency: Presets ensure a uniform design across the theme, promoting a cohesive visual identity.
+
+Scalability: They make global design changes easier during development, saving time and effort.
+
+Accessibility: Presets facilitate adherence to accessibility standards, making your patterns more usable and readable for a wider audience.
+
+- **Other tips**
+
+In the same way we remove IDs from image blocks, we need to remove queryId from query blocks too. Also, if any of our template parts have a theme attribute, that needs to remove too.
+
+`<!-- wp:query {"queryId":18,"query":{"perPage":8,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->`
+
+turns into
+
+`<!-- wp:query {"query":{"perPage":8,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true}} -->`
+
+and
+
+`<!-- wp:template-part {"slug":"header-portfolio","theme":"twentytwentyfour","area":"header","tagName":"header"} /-->`
+
+turns into
+
+`<!-- wp:template-part {"slug":"header-portfolio","area":"header","tagName":"header"} /-->`
+
+If we are constantly assigning properties to the same block over and over again (ie: border radius to images), consider moving those properties to the theme.json.
+
+When building full page patterns, let's prefix them by using page-
+
+One way to control the order in which patterns are displayed in the inserter is by changing the name of the file (they are sorted alphabetically)
+
 ### Tips for Contributors
 
 - As stated above, a goal for the theme is to have as little CSS as possible. Much of the theme's visual treatments should be handled by the Block Editor and Global Styles. As a general rule, if multiple themes would benefit from the CSS you're considering adding, it might reasonably be provided by Gutenberg instead. Let's include clear code comments for any CSS we do include.
 - Similarly, let's refrain from building any custom-built PHP or JavaScript-based workarounds for functionality that might reasonably be provided by the Block Editor, let's keep the code simple to help with future maintenance.
 - In accordance to those last two bullets, this theme has no required build process.
 - Refrain from creating pull requests for translatable strings until all patterns, parts, and templates are completed.
 
+## Discussions
+
+Starting on Wednesday 21 August 2024 at 17:00 CEST, there will be weekly Slack meetings in the `#core-themes` channel in [Make WordPress Slack](https://make.wordpress.org/chat) (registration required) to coordinate development of the theme. Agenda notes will be posted before meetings and summaries posted after the meeting.
```

Comment:
I think the time should be in UTC as that is what is used elsewhere in core where we can't change the time in userland easily for example in the sidebar of make/core and the `New Contributors Chat`.  I also think if it can be a link that would be more helpful to folks in other time zones. 

Finally, since there aren't any wednesdays between this being merged and 21 auguest, I think the part about when it starts isn't necessary. It might also be good to link to where the summaries/agendas will be posted. I assume it's make/core



```suggestion
On Wednesdays at [15:00 UTC](https://time.is/1500_in_UTC), there are weekly Slack meetings in the `#core-themes` channel in [Make WordPress Slack](https://make.wordpress.org/chat) (registration required) to coordinate development of the theme. Agenda notes will be posted before meetings and summaries posted after the meeting.
```